Classes/Systems/GithubClient.swift

```diff
@@ -46,7 +64,24 @@ struct GithubClient {
             )
         }
 
-        init(
+        static func status(
+            method: HTTPMethod = .get,
```

Comment:
Don't know if we need all the default arguments here as parameters, but just let them be as they don't harm anything really (unless someone misinterprets this and changes them, of course).